packetery/src/Packetery/Module/Checkout.php

```diff
@@ -384,6 +374,8 @@ public function updateOrderMeta( int $orderId ): void {
 			$propsToSave[ self::ATTR_CARRIER_ID ] = $carrierId;
 		}
 
+		$orderEntity = new Core\Entity\Order( (string) $orderId, $carrierId );
```

Comment:
Přesunul bych dál k místu použití. Níže je také nepoužitá proměnná $address - odstranit, pokud je zbytečná. (Tím dojde k odstranění celého cyklu a sloučení podmínek.)